animations-tutorial.md

```diff
@@ -0,0 +1,638 @@
+---
+layout: page
+title: Animations Tutorial
+permalink: /animations-tutorial/
+---
+
+Welcome to Flutter's Animations Tutorial. You may also be interested
+in our [Animations Overview](/animations/).
+
+## Introduction
```

Comment:
Hmm, scanning through the rest of this doc, it doesn't fit the 'tutorial' category. As it reads right now, this is more of a 'developer guide' (i.e. description of feature + instructions on how to use the APIs). A tutorial usually has some clear learning goals (i.e. "what you'll be able to build after following this tutorial") and a set of steps for building that type of app or adding that functionality.

My strong preference is that we do the following:

1. The API descriptions below in this doc be moved into the API docs for the respective classes (Animation, AnimationController, Tween, etc.) or the description for the [animation package](http://docs.flutter.io/flutter/animation/Animation-class.html). 

2. We revise the current [Animations Overview](/animations/) doc to be a proper developer guide. That way, we can move the information about specific animation techniques and common operations (monitoring the progress of animation, etc.) into that animations dev guide. I think it would also be nice to add some actual animated gifs in that doc to demonstrate how thse APIs work.